### ISSUE TITLE
New issue flow: Suggest posts based on title

### ISSUE THREAD

**Issue by meshulam** [codaco/feedback-site#13]
When creating an issue, I want to see potential related posts before I go through the trouble of writing a long description.

Subtask of #12

**Comment by meshulam** [comment]
This depends on #8, it will use the same (or maybe somehow more lenient) fuzzy matching. 

8 hours for rudimentary UI
